### PR TITLE
Bitcoin Core 26.0

### PR DESCRIPTION
**Pull request by fanquake**
Release annoucement for 26.0.

### CODE REVIEW

**Comment by fanquake** [comment]
Will need to update the script to stop looking for .dmg, now that we ship .zip for macOS.

File: _posts/en/posts/2023-12-06-release-26.0.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+version: 1
+
+## Only true if release announcement or security annoucement. English posts only
+announcement: 1
+
+excerpt: >
+  Bitcoin Core 26.0 is now available.
+---
+Bitcoin Core version 26.0 is now available for [download][download
+page].  See the [release notes][] for more information about the many
```
Comment (josibake) [inline _posts/en/posts/2023-12-06-release-26.0.md:19]: not sure how this markdown file works, but shouldn't this be:

```suggestion
page].  See the [release notes][release notes] for more information about the many
Comment

```

just based on how the other ones are.

Comment (fanquake) [reply _posts/en/posts/2023-12-06-release-26.0.md]: Looks like the link still works in either case (has been like this for the past release posts), but I've changed this for consistency.

**Comment by achow101** [comment]
ACK 51aa35d060f6b998551acd164e6b003e57f6f186

**Pull request merged**